xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -92,8 +106,8 @@
         <ReturnType>System.Runtime.InteropServices.Architecture</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the platform architecture on which the current app is running.</summary>
+        <value>The platform architecture on which the current app is running.</value>
```

Comment:
NIT: Should we add remarks here as we did above of the possible values? (Arm64, Arm, x64, x86)